evals/cybench/dataset.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+from typing import Callable
+
+import yaml
+
+from inspect_ai.dataset import Dataset, MemoryDataset, Sample
+
+SUPPORTED_FILE_NAMES = ["challenge.yaml", "challenge.yml"]
+
+
+def create_dataset(
```

Comment:
I'm wondering if we want to call this `cybench_dataset()` (I'm thinking we will export it for use by others so a more descriptive name will be helpful in those contexts)

Reply:
Will do!